main.go

```diff
@@ -12,5 +13,16 @@ func main() {
 		log.Fatal(err)
 	}
 
+	db, err := idb.Init()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if err = db.Connect(); err != nil {
+		log.Fatal(err)
+	}
+
+	defer db.Disconnect()
+
 	bot.Connect()
```

Comment:
Note: make sure its clear how the bot is accessing the db in future versions

Reply:
I'll be implementing the interface in a need be basis, so for now all it does is connect and disconnect from the DB. But soon it'll have a lot more things implemented!